.github/workflows/create-and-post-tag.yml

```
@@ -2,15 +2,23 @@ name: Create And Post Tag with Summary
 
 on: 
   workflow_call:
+    outputs:
+      previous_version:
+        description: "Will be used for generating release notes from the previous tag"
```

Present tense please
```suggestion
        description: "Generates release notes from comparing the previous tag to proposed tag"
```

---

Thanks for this.  I updated it for the value:
"This value is used for generating release notes from the previous tag"